TombEngine/Objects/TR1/tr1_objects.cpp

```diff
@@ -22,6 +22,7 @@
 
 // Traps
 #include "Objects/TR1/Trap/DamoclesSword.h"
+#include "Objects/TR1/Trap/TeethSpikeDoor.h"
```

Comment:
If the slot name has been changed, does it need change the file and functions names too? To SlammingDoor